pink/tasks/relative_frame_task.py

```diff
@@ -186,8 +194,19 @@ def compute_jacobian(self, configuration: Configuration) -> np.ndarray:
 
         .. math::
 
-            J(q) = \text{Jlog}_6(T_{bt}) (\text{Ad}_{T_{tr}) {}_r J_{0r}(q) -
-            \text{Ad}_{T_{tb}} {}_b J_{0b}(q))
+            J(q) = \text{Jlog}(T_{tf}) ({}_f J_{0f}(q) - \text{Ad}_{T_{fr}}{}_r J_{0r}(q))
+
+        The proof is as follow if we denote :math:`JT(q)` the left jacobian of 
```

Comment:
```suggestion
        The proof is as follow if we denote :math:`JT(q)` the left Jacobian of 
```

Reply:
Maybe we can say "local Jacobian" or "Jacobian in the local frame" to avoid saying the same things in the docs at different locations?

Otherwise it makes it harder for newcomers to learn. (If we think "left/right Jacobian" is better than "local/world" we can change it consistently throughout the docs, but I fill the latter is somehow clearer?)

Reply:
Yes !